src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Microsoft.DotNet.UpgradeAssistant.Cli.csproj

```diff
@@ -91,6 +91,9 @@
     <Extension Include="..\..\extensions\default\Microsoft.DotNet.UpgradeAssistant.Extensions.Default\Microsoft.DotNet.UpgradeAssistant.Extensions.Default.csproj">
       <Name>default</Name>
     </Extension>
+    <Extension Include="..\..\extensions\vb\Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic\Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic.csproj">
```

Comment:
Should this be `Microsoft.DotNet.UpgradeAssistant.Extensions.VisualBasic.csproj` to follow the other patterns?